js/controller.js

```diff
@@ -1000,6 +1000,7 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       this.mb.publish(OO.EVENTS.SEEK, seconds);
       if (this.state.screenToShow == CONSTANTS.SCREEN.END_SCREEN) {
         this.state.pauseAnimationDisabled = true;
+        this.endSeeking();
```

Comment:
to ensure playhead is rendered correctly when transitioning from end screen to pause screen